cssproject.html

```diff
@@ -0,0 +1,127 @@
+				h1{
+					line-height: 40px;
+					font-family: "Monotype Corsiva",cursive;
+					font-size: 60px;
+					margin-bottom: 1px;}
+				h2{
+					font-family: courier, monospace;
+					line-height: 0px;
+					font-weight: lighter;
+					font-size: 12px;
+					padding-left: 50px;}
+				h3{
+					font-family: arial, sans-serif;
+					font-size: 45px;
+					margin-bottom: 1px;
+				}
+				h4{
+					font-weight: lighter;
+					line-height: : 0px;
+					margin-top: 0px;
+					margin-bottom: 20px;
+				}
+				p.footer{
+					padding-left: 80px;
+					font-size: 15px;
+				}
+				p.steps1{
+					text-indent: 20px;
+					font-size: 15px;
+				}
+				p.steps2{
+					text-indent: 20px;
+					font-size: 15px;
+				}
+				p.steps3{
+					text-indent: 20px;
+					font-size: 15px;		
+				}
+				p.checklist{
+					font-family: courier;
+					background-image: url("/home/jon/CodeFellows/CSSproject/list-bg.png")
+					font-size: 15px;
+				}
+				p.topdirectbar{
+					padding-right: 310px;
+					text-align: right;
+				}
+				p.printico{
+					padding-right: 340px;
+					text-align: right
+				}
+				p.imagevan{
+					text-align: left;
+				}
+				p.imagesmall{
+					text-align: center;
+				}
+				}
+			</style>
+	</head>
+	<body>
+		
+		<h1><img src="logo.png" href="logo" align="top">Delicious<br /><h2>THE BEST FOOD BLOG ON THE WEB.</h2></h1>
+		<p class="topdirectbar">
+		<img src="fb-icon.png" href="fb-icon">
+		<img src="twit-icon.png" href="twit-icon">
+		<img src="gp-icon.png" href="gp-icon">
+		<img src="insta-icon.png" href="insta-icon">
+		<img src="flic-icon.png" href="flic-icon">
+		<img src="pint-icon.png" href="pint-icon">
+		<img src="mail-icon.png" href="mail-icon"></br>
+		</p>
+		<img src="borderslash.png" href="borderslash">
+		<h3>Chocolate Pizza</h3>
+			<h4>post on 15 Dec 2013/Desserts</h4>
+			<p class="printico">
+			<img src="print-icon.png" href="print-icon.png">
+			</p>
+			<img src="choco-pizza.png" href="choco-pizza">
+			
+
+				
+			
+				<P class="steps1">For the fig-swirl: Melt butter over medium heat in a saucepan with water and lemon juice. Cook over medium heat,sitrring frequently, until you have a chunky jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+				<p class="steps2">Ice cream:In a small pot over medium heat, combine milk, and granulated sugar until sugar is competely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until te fig mixture is cooled.</p>
+				<p class="steps3">Using an ice cream machine, pour liquids into the forzen basin and process according to manufacturer isntructions, i.e., let spin and thinken for 20 minutes before adding marscapone,fig jam mixture, and the nuts. Continue to process for +/10 minutes. Pour semi frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+				<img src="list-bg.png" href="list-bg">
+
+				<p class ="checklist">
```

Comment:
You are using the p tag too generally. Please review the use of the p tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p